app/models/api/transliterate.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+// Check that the class exists before trying to use it
+if (! class_exists('Transliterator')) {
+    return $json = ['error' => 'class Transliterator not available'];
```

Comment:
I think you can set $request->error then use this function here https://github.com/mozfr/transvision/blob/master/app/classes/Transvision/API.php#L309